src/main/java/SYSC4806Project/ItemQuantityList.java

```diff
@@ -101,7 +104,7 @@ public int getItemQuantity(Product product) {
     }
 
 
-    public class ItemQuantityPair {
+    public static class ItemQuantityPair {
```

Comment:
IntelliJ forced me to make this static when I was writing the test classes. Any objections?

Reply:
I don't think there is any problems doing it this way.